app/models/user.js

```diff
@@ -1,41 +1,22 @@
 import Model, { belongsTo, attr } from '@ember-data/model';
-import { computed } from '@ember/object';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  firstName: attr('string'),
-  lastName: attr('string'),
-  emailLink: attr(),
-  phoneLink: attr(),
-  account: belongsTo('account'),
-  group: belongsTo('account-group', {
-    inverse: null,
-  }),
-  organization: belongsTo('organization'),
+export default class User extends Model {
+  @attr('string') firstName;
+  @attr('string') lastName;
+  @attr('email') emailLink;
+  @attr('phone') phoneLink;
 
-  email: computed('emailLink', {
-    get() {
-      return this.emailLink && this.emailLink.replace(/^mailto:/, '');
-    },
-    // eslint-disable-next-line no-unused-vars
-    set(key, value) {
-      this.set('emailLink', `mailto:${value}`);
-      return value;
-    },
-  }),
-  phone: computed('phoneLink', {
-    get() {
-      return this.phoneLink && this.phoneLink.replace(/^tel:/, '');
-    },
-    // eslint-disable-next-line no-unused-vars
-    set(key, value) {
-      this.set('phoneLink', `tel:${value.replace(/[/ .]/, '')}`);
-      return value;
-    },
-  }),
+  @belongsTo('account') account;
+  @belongsTo('account-group', { inverse: null }) group;
+  @belongsTo('organization') organization;
 
-  fullName: computed('firstName', 'lastName', function () {
+  get email() {
+    return this.emailLink;
+  }
+  get phone() {
+    return this.phoneLink;
+  }
```

Comment:
Can you just rename the `{email,phone}Link` properties to `{email,phone}` in `mu-cl-resources` to get rid of these aliases?